scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -709,14 +709,14 @@ class ScalaTestAntTask extends Task {
       throw new BuildException(
         "reporter type 'reporterclass' requires 'classname' attribute")
 
-    args += reporter.getClassName
+    args += reporter.getClassName.nn
   }
 
   /**
    * Sets value of the <code>runpath</code> attribute.
    */
   def setRunpath(runpath: Path): Unit = {
-    for (element <- runpath.list) {
+    for (element <- runpath.list.map(_.nn)) {
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.